### ISSUE TITLE
ui5-icon: empty aria-label should not be present

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#1282]
**Describe the bug**
There is an empty aria-label rendered on svg level, when no accessibleName is provided.

**To reproduce**
Steps to reproduce the behaviour:
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Icon/
2. Inspect the first icon, observe the svg element. There is an aria-label control with no value - it should be removed.
